src/Evolution/Initialization/Tags.hpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/IndexType.hpp"
+
+/// \cond
+template <size_t VolumeDim, typename TargetFrame>
+class Domain;
+/// \endcond
+
+namespace Initialization {
+namespace Tags {
+
+template <size_t Dim>
+struct InitialExtents : db::SimpleTag {
+  static std::string name() noexcept { return "InitialExtents"; }
+  using type = std::vector<std::array<size_t, Dim>>;
+};
+
+template <size_t Dim>
```

Comment:
why are there separate Initialization Tags? And in particular for Domain?

Reply:
FYI: I move them to `Domain/Tags.hpp` and `Evolution/Tags.hpp` in #1575

Reply:
I wanted to have it separate since this will need to be cleaned up once the Domain is in the global cache anyway.